lua/kubectl/utils/tables.lua

```diff
@@ -243,7 +243,7 @@ local function addDividerRow(divider, hints, marks)
     local count = divider.count or ""
     local filter = divider.filter or ""
     local info = resource .. count .. filter
-    local padding = string.rep("—", half_width - math.floor(#info / 2))
+    local padding = string.rep("-", half_width - math.floor(#info / 2))
```

Comment:
Is this a different kind of unicode or a regular dash?

Reply:
the current one is some unicode special one which I thought was nice in https://github.com/Ramilito/kubectl.nvim/commit/e0bfb6eda2248e5f4ef83cf35cca44e95645e253#diff-c7fc12e395ddb005b4efffd49c7bf397d0b39048c10e57da1cf19b12b043287fR284

but it doesn't parse it as a ligature when I switched to Wezterm so I'm reverting to a simple plain dash (`-`)